src/components/Navigation/Navigation.tsx

```diff
@@ -8,10 +8,10 @@ const Navigation: FC = ({ children }) => {
   const classes = useStyles({});
 
   return (
-    <AppBar position='static'>
+    <AppBar position="static">
```

Comment:
Can we update this the prettier rule to make this single quote?

Reply:
I would actually prefer a `double quoted string` on `jsx elements` as it visualize that it is an `element`. Coming from `html` standpoint. I like to treat `jsx elements` as real `super powered html elements` so there is distinction between an `element property` vs `variable` (which uses single quote). And for me it's easier to work with using `double quotes` on `jsx properties` when tackling with `'` apostrophe so we wouldn't have to use unnecessary escape `\`.

i.e 
```typescript
<input placeholder="Can't" />
// vs
<input placeholder={'Can\'t'} /> // lot of work
```

Reply:
Honestly I would just use string template if so
```
<input placeholder={`Can't`} /> // lot of work
```
But as long as we set this value in prettier config, I am okay with keeping double quote for property values.